index.html

```diff
@@ -338,11 +339,13 @@ <h1 class="first-title-3">Contact me</h1>
             required
           />
           <input
+            id="email"
             type="email"
             name="email-address"
             placeholder="Email address"
             required
           />
+          <small class="error"></small>
```

Comment:
- [ ] You have done well implementing the form validation. Currently, the error message shows up close to the `email input` but it is required that it shows up close to the submit button. Let's make the error message show up close to the "submit" button to get all ⭐

Reply:
I have modified the readme file but on the main branch.

Reply:
Please modify it in this branch 🙏

Reply:
@omar25ahmed 
Please let's modify the README in this branch instead of the main branch. This is a requirement to have a descriptive README on every pull request you created. Please work on this as I wait for you to complete it. You have only one attempt left. Please I have your pull request to review again and you haven't implemented the requested changes.